src/vfs/createNodeFileSystem.ts

```diff
@@ -0,0 +1,64 @@
+import { VirtualFileSystem, FileStat } from "./virtualFileSystem";
+import fs from "fs";
+import path from "path";
+
+export function createNodeFileSystem(
```

Comment:
In Misti, we use docstring-style comments for all public API functions that users interact with. These comments are used to generate documentation hosted at https://nowarp.io/tools/misti/api/.

Please add docstrings here and to all added public methods/functions.

Reply:
Also, are there any requirements to the input arguments? e.g. in Tact the path passed to VFS constructors has to be absolute.

Reply:
There is no change in existing flow. You can pass relative as well absolute contract path to the CLI. In VFS it will take current path as a base/root path where it will be executed.